### DISCUSSION TITLE
Is there any example of using server rendering with rspack?

### DISCUSSION THREAD

**Discussion by sowtame** [web-infra-dev/rspack#5874] [Core]
server side rendering with express for example with hot reload

**Comment by xc2** [comment]
i happened to make a small example: https://github.com/xc2/rspack-usecases/tree/main/rspack/loadable

**Comment by lzxb** [comment]
i am using Vue2 for SSR

**Comment by lzxb** [comment]
see here
https://github.com/dp-os/gez/tree/feat-v4/packages/rspack-vue2
https://github.com/dp-os/gez/tree/feat-v4/examples/ssr-rspack-vue2
